BusinessLayer/SongPlayer/SongPlayer.cpp

```diff
@@ -209,24 +211,33 @@ void SongPlayer::updateInfo()
     emit updateGUI(title_, artist_, img);
 }
 
-QColor SongPlayer::getColor(QImage img)
+QColor SongPlayer::getColor(QImage img, int number)
```

Comment:
Since this is a fairly complex process, let's leave a comment explaining what's happening for the future maintainers